packages/cms/pages/settings/team-members/team-members.tsx

```diff
@@ -68,18 +77,162 @@ const UserRole = styled.span`
 `
 
 interface ProfileProps {}
+interface FlyOutProps {
+  setIsFlyoutVisible: (b: boolean) => void
+  selectedMember: TeamMemberProps
+  showChangeMemberRoleModal: () => void
+  getTeamMembers: () => void
+}
+const FlyOut: React.FC<FlyOutProps> = ({
+  setIsFlyoutVisible,
+  selectedMember,
+  showChangeMemberRoleModal,
+  getTeamMembers
+}) => {
+  const simpleFlyoutTitleId = useGeneratedHtmlId({
+    prefix: 'simpleFlyoutTitle'
+  })
+
+  const [memberDetails, setMemberDetails] = useState({
```

Comment:
We have the `useForm` hook you can use for forms. Takes care of errors, state and all.